app/assets/javascripts/tylium/pages/activities.js

```diff
@@ -29,4 +29,20 @@ document.addEventListener('turbolinks:load', function() {
       }
     });
   }
+
+  $(function() {
+    $('[data-behavior~=activity-filters] select').change(function () {
+      $(this).closest('form').submit();
+    });
+
+    $('[data-behavior~=activity-filters] input[type="date"]').change(function () {
+      const dateValue = $(this).val();
+      const periodStart = $('[data-behavior~=period-start]').attr('min');
```

Comment:
The variable name is a bit misleading since this isn't the value of the periodStart but the earliest date that's possible, which I believe is hardcoded to be 01-01-2016.

Reply:
The default date is just the fallback date that is hardcoded in the view (01-01-2016) . This variable still refers though to the actual periodStart. When a user changes the starting date the hardcoded date gets overwritten.